gemfiles/activesupport_6.1.gemfile

```diff
@@ -0,0 +1,12 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "activesupport", "6.1"
+
+group :test do
+  gem "webmock"
+  gem "simplecov", require: false
+end
```

Comment:
Can we put these into development dependencies and remove from here?

other projects I worked on only had real dependencies here. (e.g.: activesupport, faraday)

Reply:
yeah, they were in the Gemfile.  I can put them in the dev dependencies

Reply:
Great idea, done.